.azure-pipelines/linux-analyzer-test.yml

```diff
@@ -118,7 +118,7 @@ jobs:
       gradleOptions: '-Xmx8192m'
 
   # Clean disk space before caching starts, otherwise the disk might run full.
-  - bash: ./gradlew --no-daemon clean
+  - bash: ./gradlew --console=plain --no-daemon clean
```

Comment:
Could you add to the commit message what happens when not doing so? Does Gradle print ANSI sequences? Or line-by-line progress that clutters the output?

Reply:
It looks like Gradle automatically detects the correct console type, I'll replace this with a commit that instead removes the option from the other Gradle tasks.

Reply:
Thanks, that's what I was trying to imply :wink:

Reply:
Done.